posixlib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -92,14 +93,21 @@ object unistd {
   def fdatasync(filedes: CInt): CInt = extern
 
   def fexecve(fd: CInt, argv: Ptr[CString], envp: Ptr[CString]): CInt = extern
+  def _Fork(): pid_t = extern
   def fork(): pid_t = extern
   def fpathconf(fd: CInt, name: CInt): CLong = extern
+
   @blocking
   def fsync(fildes: CInt): CInt = extern
+
   @blocking
   def ftruncate(fildes: CInt, length: off_t): CInt = extern
 
   def getcwd(buf: CString, size: CSize): CString = extern
+
+  // No @blocking necessary. Will block only very early in system boot.
+  def getentropy(buffer: Ptr[CVoidPtr], length: size_t): CInt = extern
```

Comment:
Does `CVoidPtr` have to be wrapped in `Ptr`. I didn't actual know we had that type. How is that used? I think we produce Scaladoc for these 2 if we need documentation specific to Scala Native usage - the POSIX docs or unix manuals are fine otherwise. I guess I need to change the main page to point to version `0.5` https://javadoc.io/doc/org.scala-native/posixlib_native0.5_3/latest/scala/scalanative/posix/unistd$.html

Reply:
Good catch, thank you.   Fixed now.

You are right, the `CVoidPtr` does not need to be wrapped in `Ptr[]`.  Old muscle memory dies
hard.

I think `CVoidPtr` was introduced six months or a year ago to exactly deal with situations
like this and to be clearer than the previous `Ptr[Byte]` idiom.  A worthwhile change and
an definite improvement in my book (once I adjust to the new idiom).

Reply:
Nice, I missed that change and it is much better to read.